src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -374,6 +374,42 @@ class LSFJobManager extends BatchEuphoriaJobManagerAdapter {
         cacheLock.unlock()
     }
 
+    @Override
+    String getJobIdVariable() {
+        return "LSB_JOBID"
+    }
+
+    @Override
+    String getJobNameVariable() {
+        return "LSB_JOBNAME"
+    }
+
+    @Override
+    String getQueueVariable() {
+        return 'LSB_QUEUE'
+    }
+
+    @Override
+    String getJobArrayIndexVariable() {
+        return "LSB_JOBINDEX"
+    }
+
+    @Override
+    String getNodeFileVariable() {
```

Comment:
What is a node file?

Reply:
Here I added variables from PBS. Could be that not all are important. This is a candidate for a less important one. I can remove it. I actually decided not to include all variables anyway, becaus not all variables can trivially be mapped semantically between PBS and LSF (not checked SGE and SLURM -- probably similar) and some have no reasonably equivalent (also for direct execution).